### ISSUE TITLE
PD Resource library

### ISSUE THREAD

**Issue by ajp9e** [castl350/AEII#8]
When Ann was reviewing the website on her phone she noted this:

_PD Resource Library doesn't work for me to see search results on my cell phone (not "responsive" or whatever the technical term is)_

![Screenshot_20200410-084746](https://user-images.githubusercontent.com/19825720/78997559-d1f0c100-7b14-11ea-83c7-30300729fa6b.png)

**Comment by kneesen22** [comment]
yes..it's in an iframe and not something we'll be able to address probably.
